### PR TITLE
Use conductor copy nwjs + update name

### PR DESCRIPTION
**Pull request by flebel**

### CODE REVIEW

File: buildScripts/deployArtifacts.sh [hunk 1]
```diff
@@ -3,13 +3,20 @@
...
-aws s3 cp ./ s3://${AWS_S3_BUCKET_NAME}/companion --recursive --acl public-read 
+aws s3 cp ./ s3://${AWS_S3_BUCKET_NAME}/companion --recursive --acl public-read
 
+# Upload build as `latest` version.
 if [ "$CI_BRANCH" == "master" ]; then
     version="$(jq -r .version ../build/package.json)"
     echo "$version @ `date`" > current-version.txt
+
+    # Update the latest version.
+    mv companion*dmg companion-latest-osx-installer.dmg
```
Comment (hoolymama) [inline buildScripts/deployArtifacts.sh:16]: I think this is good - but I think another change needs to happen in make installers.sh - I'll comment down there.

File: buildScripts/makeInstallers.sh [hunk 2]
```diff
@@ -16,11 +16,10 @@ else
     version="${version}-dev-$(echo $CI_BRANCH | tr '/' '-')"
```
Comment (hoolymama) [inline buildScripts/makeInstallers.sh:16]: Currently, when we build the installers, we take the version from package.json and use it in the `builder` command, so as to override whatever is in the XML file.  I wrote a conditional there to set the version to **latest** for the master branch, but I think that's wrong. The reason is that after a customer runs the installer, they get the word latest in the installed app filename. **/Applications/companion-latest/companion.app**

So I think this code should go.

```
if [ "$CI_BRANCH" == "master" ]; then
    version=latest
else
    version="${version}-dev-$(echo $CI_BRANCH | tr '/' '-')"
fi
```

However think what you did in deployArtifacts.sh (rename the built file) is correct. But if the builder command uses only the version string, then we may want to add back in the branch name (for non master branches) when you rename the files.

All this is on the assumption that the desired behavior is:

- Customer downloads **companion-latest-osx-installer.dmg**
- When they click the installer, it creates **/Applications/companion-0.5.2/companion.app** 
- during the install process, the prompts will say things like "Install version 0.5.2 of the companion app?"

Currently, when they install, they get **/Applications/companion-latest/companion.app**

Comment (flebel) [reply buildScripts/makeInstallers.sh:16]: I'm not following you.

> However think what you did in deployArtifacts.sh (rename the built file) is correct. But if the builder command uses only the version string, then we may want to add back in the branch name (for non master branches) when you rename the files.

The `mv` rename only happens for builds from the `master` branch. The branch name stays in the filename for any other builds.

What is the issue to be resolved? Is it the `-master` appended at the time of installation? Can we leave the version blank to address that?

Comment (hoolymama) [reply buildScripts/makeInstallers.sh]: Hi @flebel, 

See the commits below. What I was getting at (and explaining badly), is that the version string in the file `makeInstallers.sh` is used throughout the installation process. For example: 

**Would you like to install Conductor companion version 0.2.5-dev-fix-the-thing-that-broke?**

That would also be the installed folder name.

So I was suggesting to keep it a simple semver in `makeInstallers.sh` and only change it for the name of the deployed installer files, in `deployArtifacts.sh`

**Pull request merged**